sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -279,27 +254,6 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
       structDf.select(hash($"a", $"record.*")))
   }
 
-  test("Star Expansion - explode should fail with a meaningful message if it takes a star") {
```

Comment:
Can we rewrite this test case using select(explode()), like what we did in the following test cases?

Reply:
Yeah I'll try to bring back to the test case.